team-01/src/main/java/ubc/cosc322/MonteCarloPlayer.java

```diff
@@ -130,5 +128,23 @@ public int getVisits() {
     public AmazonsAction getAction() {
       return action;
     }
+
+    // TODO
+    private double getUCB() {
+      // EXPLORATIONS = constant defined at the top of the class.
+      if (this.getVisits() == 0){
+        return EXPLORATIONS;
+      }
+
+      // uct = v = total score / number of visits == avg value of the state.
+      float uct =  this.wins / this.getVisits();
```

Comment:
I would make this consistent from an OOP perspective: either use a getter function for both values, or use the local attribute value for both:
```java
float uct =  this.wins / this.visits; // option 1
float uct =  this.getWins() / this.getVisits(); // option 2
```